servicenow/README.md

```diff
@@ -0,0 +1,205 @@
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+
+distributed under the License is distributed on an "AS IS" BASIS,
+
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+
+limitations under the License.
+
+
+# **ServiceNow Integration**
+
+
+## 
+**Setup**
+
+
+### 
+**Prerequisites**
+
+
+
+*   Follow the instructions on the [main README file](https://github.com/GoogleCloudPlatform/dialogflow-integrations#readme) in the root directory of this repository.
+*   Create a [ServiceNow Developer account](https://developer.servicenow.com/dev.do).
+*   Create a [Dialogflow Agent](https://dialogflow.cloud.google.com/).
+
+### 
+**Configuring ServiceNow**
+
+*   Log into your ServiceNow Developer account.  If this is the first time you are logging in, you will need to create an _Instance_.  Follow the instructions on creating an Instance and select a _Release_.  As of this writing, Paris is the latest Release and it is recommended that you choose one of the latest releases, Orlando or Paris for this integration.
+*   Click on your Instance URL (ex. _https://dev10101.service-now.com/_) and log in.  You will now be logged into the admin console.
+*   From the Navigation menu, search for “Scripted REST APIs” and select this option.
+*   From the Scripted REST APIs menu, click on the “New” button.
+*   Provide a descriptive name like “Dialogflow Post”.  The API ID field will be automatically populated after you provide a name.  Click on the “Submit” button.
+
+
+
+
+
+![alt_text](images/snow-rest-api.png "image_tooltip")
+
+
+
+
+*   You will be returned to the “Scripted REST APIs” menu.  From the list of Services, find and select the one you just created.
+*   **_Make a note of the “Base API path” as you will use this value to make your Dialogflow Webhook._**
+
+
+
+
+
+![alt_text](images/snow-base-api.png "image_tooltip")
+
+
+
+
+*   Scroll down the page and on the Resources tab, click on the “New” button.
+
+    
+
+
+
+![alt_text](images/snow-rest-resource.png "image_tooltip")
+
+
+*   On the “Scripted REST Resource” page, provide a descriptive Name and in the “HTTP method” field, select “POST”.
+
+
+
+
+
+![alt_text](images/snow-rest-name.png "image_tooltip")
+ 
+
+
+
+*   In the “Script” section, replace the code with the code in the file “servicenow-script.txt” and then click the “Submit” button at the top.
+
+### 
+**Configuring Dialogflow**
+
+*   In the [Dialogflow Console](https://dialogflow.cloud.google.com/), click on "Intents" and then click on the “Create Intent” button.  We will create the first intent to create an incident/ticket in ServiceNow.
```

Comment:
May I ask the reason why didn't you create an agent template and export to this directory? So customer will only need to download that zip file then export it into their own agent?

Reply:
I didn't want to assume the user is familiar with Dialogflow, so I wanted them to learn how to configure Dialogflow agent as well as ServiceNow.

Reply:
I think this may makes the setup process more complicated. Instead, we could point them to Dialogflow get started guide to set up the agent, that may make things simpler. Thoughts?

Reply:
Yes, it definitely makes setup longer.  Let's go with your idea as it follows the salesforce repo and I'll provide the agent zip and remove the agent setup from the readme.  I will still have my internal doc with full instructions that I can share with CEs and others by request.

Reply:
updated the Readme and added the agent zip file